src/bin/eic_curl_authorized_keys

```diff
@@ -87,7 +87,7 @@ if [ "${id_exit}" -ne 0 ] ; then
 fi
 
 # Verify that we have active keys.  Fast-exit if we do not.
-keys_status="$(/usr/bin/curl -s -f -m 1 -H "${IMDS_TOKEN_HEADER}" -o /dev/null -I -w %{http_code} "${IMDS}/managed-ssh-keys/active-keys/${1}/")"
+keys_status="$(/usr/bin/curl -s -f -m 1 -H "${IMDS_TOKEN_HEADER}" -o /dev/null -I -w %{http_code} "${IMDS}/managed-ssh-keys/active-keys/${1}/")" && true
```

Comment:
You should be able to get the same effect by removing `-f` which is what causes `curl` to check the HTTP status code and exit 22 if it's not a success status:

> (HTTP)  Fail silently (no output at all) on server errors. This is mostly done to better enable scripts etc. to better deal with failed attempts. In normal cases when an HTTP server fails to deliver  a  document,  it returns  an HTML document stating so (which often also describes why and more). This flag will prevent curl from outputting that and return error 22.